### ISSUE TITLE
Demo Does Not run with Tensorflow Version 2.0

### ISSUE THREAD

**Issue by dbl001** [uclnlp/gntp#1]
$ ipython
Python 3.6.7 | packaged by conda-forge | (default, Nov  6 2019, 16:19:42) 
Type 'copyright', 'credits' or 'license' for more information
IPython 7.11.1 -- An enhanced Interactive Python. Type '?' for help.

In [1]: import tensorflow                                                       

In [2]: print(tensorflow.__version__)                                           
2.0.0

In [3]: import tensorflow.contrib                                               
---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
<ipython-input-3-9ce6c8f43c36> in <module>
----> 1 import tensorflow.contrib

ModuleNotFoundError: No module named 'tensorflow.contrib'

**Comment by pminervini** [comment]
Yep it's currently a problem - our code is still based on TF 1.x

I can start porting it to TF 2.x in the second half of February

**Comment by 570710065-jili** [comment]
TF 1.x can't run on cuda11.

**Comment by Yanzhiwen001** [comment]
Hi, did you managed to transfer it to TF2.x? I also have the problem with higher cuda version on GPU .
